src/components/AboutUs/StaffCard.js

```diff
@@ -58,9 +58,16 @@ const SocialLinks = ({ data }) => {
   ) : null
 }
 
-const StaffCard = ({ name, image, role, description, socialLinks = [] }) => {
+const StaffCard = ({
+  name,
+  image,
+  role,
+  description,
+  socialLinks = [],
+  ...props
+}) => {
   return (
-    <Col width={[1, 1, 1, 1, 6 / 12, 4 / 12]}>
+    <Col width={[1, 1, 1, 1, 6 / 12, 4 / 12]} {...props}>
```

Comment:
Can we add the `data-testid` explicitly instead of spreading the rest of the props, seems a bit dangerous